spec.go

```diff
@@ -276,17 +276,22 @@ type Source struct {
 	Excludes []string `yaml:"excludes,omitempty" json:"excludes,omitempty"`
 }
 
+type PackageConstraints struct {
+	Version []string `yaml:"version,omitempty" json:"version,omitempty"`
```

Comment:
general question, here version been an array, does it make Dalec able to handle install of multiple versions of same package?

Reply:
Can we add a comment noting that these are version constraints?

Reply:
This is done.